sympy/matrices/tests/test_commonmatrix.py

```diff
@@ -522,6 +522,14 @@ def test_replace_map():
     assert N == K
 
 
+def test_rot90():
+    A = Matrix([[1, 2], [3, 4]])
+    assert A == A.rot90(0)
+    assert A.rot90(1) == A.rot90(-3)
```

Comment:
test the actual values here, too
```python
A.rot90() == A.rot90(-7) == A.rot90(-3) == Matrix(((2, 4), (1, 3)))
```

Reply:
it'd have to be 😄 
```A.rot90() == A.rot90(-7) == A.rot90(-3) == Matrix(((3, 1), (4, 2)))```

Reply:
you may have to pull the branch from the github to your local branch, or conflict can occur

Reply:
I did that 😄 ... thanks btw!